sc/src/test/java/uk/ac/ebi/atlas/metadata/CellMetadataDaoIT.java

```diff
@@ -147,6 +150,24 @@ void invalidCellIdAndExperimentAccessionHasNoFactorFields() {
         assertThat(subject.getFactorFieldNames(experimentAccession, cellId)).isEmpty();
     }
 
+    @Test
+    void experimentWithMissingValuesReturnsNotAvailable() {
+        String experimentAccession = "E-GEOD-71585";
+
+        ImmutableList<String> cellIdsWithMissingValues = ImmutableList.of(
+                "SRR2138737",
+                "SRR2140225",
+                "SRR2139550",
+                "SRR2139566");
```

Comment:
As a future improvement these could be randomly sampled from Solr. Could you add a TODO comment?